internal/tracegen/.nocover

```diff
@@ -0,0 +1 @@
+Test utility
```

Comment:
per offline discussion, this is worth merging even though it doesn't have tests (it is itself a test utility).